R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1177,6 +1177,9 @@ test_that("column functions", {
   c17 <- cov(c, c1) + cov("c", "c1") + covar_samp(c, c1) + covar_samp("c", "c1")
   c18 <- covar_pop(c, c1) + covar_pop("c", "c1")
   c19 <- spark_partition_id()
+  c20 <- to_timestamp(c) + trim(c) + unbase64(c) + unhex(c) + upper(c)
+  c21 <- to_timestamp(c, "yyyy") + trim(c) + unbase64(c) + unhex(c) + upper(c)
+  c22 <- to_date(c, "yyyy") + trim(c) + unbase64(c) + unhex(c) + upper(c)
```

Comment:
you probably don't need to duplicate `+ trim(c) + unbase64(c) + unhex(c) + upper(c)` - as explained on the other comment, the goal (from what I can see) is to call each of these once.

So you could condense these 3 lines into 
```
c20 <- to_timestamp(c) + to_timestamp(c, "yyyy") + to_date(c, "yyyy")
```